### PR TITLE
ci: use Deno v2 for a test running for deno

### PR DESCRIPTION
**Pull request by nakasyou**
Deno 2 has been released, so I changed a CI.

### The author should do the following, if applicable

- [ ] Add tests
- [ ] Run tests
- [ ] `bun run format:fix && bun run lint:fix` to format the code
- [ ] Add [TSDoc](https://tsdoc.org/)/[JSDoc](https://jsdoc.app/about-getting-started) to document the code

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: denoland/setup-deno@v1
```
Comment (yusukebe) [inline .github/workflows/ci.yml:77]: We can use [`denoland/setup-deno@v2`](https://github.com/denoland/setup-deno).

**Comment by yusukebe** [comment]
Hi @nakasyou 

Thank you! I've left a comment.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (nakasyou) [reply .github/workflows/ci.yml]: I changed it.

**Comment by nakasyou** [comment]
I have a question. Deno 2.0 has breaking changes. Should I test both 1.x and 2.0?

**Comment by yusukebe** [comment]
@nakasyou 

Thank you for the fix.

> I have a question. Deno 2.0 has breaking changes. Should I test both 1.x and 2.0?

Hmm. I think it's unnecessary. It's enough to test the latest major version, and if there are problems, we can say please try to use the latest Deno.

@nakasyou What is your thought?

**Comment by nakasyou** [comment]
@yusukebe 
>  It's enough to test the latest major version

I agree. Also, [Deno's LTS](https://docs.deno.com/runtime/fundamentals/stability_and_releases/#long-term-support-(lts)) is shorter than Node.js, so there is no need to test multiple versions.

**Comment by yusukebe** [comment]
@nakasyou 

Okay. I'll merge this now. Thanks!

**Pull request merged**